client/actions/agenda.js

```diff
@@ -35,6 +35,42 @@ const createAgenda = ({ name }) => (
 )
 
 /**
+ * Action Dispatcher for updating an Agenda
+ * @param {string} _id - The id of the Agenda
+ * @param {string} name - The new name of the Agenda
+ * @return arrow function
+ */
+const updateAgenda = ({ _id, name }) => (
```

Comment:
`updateAgenda` definition looks very similar to `createAgenda`. I would factorize them in order to avoid repetition in the triggered actions. Because in the future if we have to change them somewhere, we would probably have to do it also the other one.